tools/topology/topology2/platform/intel/bt-generic.conf

```diff
@@ -192,12 +192,12 @@ Object.Pipeline {
 }
 
 Object.Base {
-	route."5" {
+	route."500" {
```

Comment:
@ranj063 Do we have any better best-known-method to allocate the route ids?

Reply:
@kv2019i not at this time but I have a pending PR to avoid this https://github.com/alsa-project/alsa-utils/pull/188. Then all included conf files can start from 1 for their route IDs and the compiler will make them unique